### DISCUSSION TITLE
Wake on Lan

### DISCUSSION THREAD

**Discussion by chape87** [unknownskl/xbox-xcloud-client#897] [Ideas]
Outside of microphone support, one other key piece to take the app to the next level is wake on lan. Currently i have to turn the xbox on manually, which is on another floor. If there was a button in the gui to send the magic packet and wake up the xbox, itd be great.

**Comment by rackerby** [comment]
Is your console power option set to "Sleep" or "Shutdown"?

**Comment by anthonyylee** [comment]
Perhaps try changing the Xbox console power setting as if you would for a normal remote play.

![image](https://github.com/unknownskl/greenlight/assets/9303869/0f3d7e72-722a-4c66-8401-2499dea138fa)
(Src: [Xbox remote play instructions](https://support.xbox.com/en-US/help/games-apps/game-setup-and-play/how-to-set-up-remote-play))
